DOCUMENTATION.md

```diff
@@ -81,6 +82,73 @@ You need a way to identify your users to provide the same variants through Split
 
 We can now use this value for `userId` when initialising the Splitster client.
 
+### Running the experiments against a segment of users
+
+So far, the tests run for every user. Sometimes, you want to run the experiments against only a subset of your users, say for example only against English or French speaking users. This is where the `user` configuration option of the client comes into play.
+
+This option acts as a filter on the tests provided to the client, to select which tests should be active for this specific user. Let's see an example.
```

Comment:
really nice description, maybe just some styling suggestions to the code example below: